source/images/homepage/transparency.svg

```diff
@@ -0,0 +1,53 @@
+<svg class="LogoTransparency" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 302 302" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>
+    transparency
+  </title>
+  <defs>
+    <linearGradient id="transparency-linearGradient-1" x1="50%" x2="50%" y1="0%" y2="100%">
+      <stop stop-color="#09DFF3" offset="0%"/>
+      <stop stop-color="#44BEFF" offset="100%"/>
+    </linearGradient>
+    <circle id="transparency-path-2" cx="150" cy="150" r="150"/>
+    <circle id="transparency-circle-path" cx="150" cy="150" r="150"/>
+    <linearGradient id="transparency-linearGradient-6" x1="50%" x2="50%" y1="0%" y2="62.304%">
+      <stop stop-color="#09DFF3" offset="0%"/>
+      <stop stop-color="#44BEFF" offset="100%"/>
+    </linearGradient>
+  </defs>
+  <g fill="none" fill-rule="evenodd">
+    <g>
+      <g transform="translate(1 1)">
+        <mask id="transparency-mask-3" fill="#fff">
+          <use xlink:href="#transparency-path-2"/>
+        </mask>
+        <use fill="#EBF5FA" stroke="url(#transparency-linearGradient-1)" xlink:href="#transparency-path-2"/>
+        <g mask="url(#transparency-mask-3)">
+          <mask id="transparency-mask-5" fill="#fff">
+            <use xlink:href="#transparency-circle-path"/>
+          </mask>
+          <use fill="#EBF5FA" xlink:href="#transparency-circle-path"/>
+          <g stroke="#FFF" mask="url(#transparency-mask-5)">
+            <path d="M-20.342 153.15L261.44-15.88" stroke-linecap="square"/>
+            <path d="M-20.342 187.15L261.44 18.12" stroke-linecap="square"/>
+            <path d="M320.44 153.15L38.658-15.88" stroke-linecap="square"/>
+            <path d="M297.44 221.15L15.658 52.12" stroke-linecap="square"/>
+            <path d="M290.44 251.15L8.658 82.12" stroke-linecap="square"/>
+            <path d="M297.44 173.15L15.658 4.12" stroke-linecap="square"/>
+            <path d="M320.44 147.12L38.658 316.15" stroke-linecap="square"/>
+            <path d="M320.44 113.12L38.658 282.15" stroke-linecap="square"/>
+            <path d="M-19.342 147.12L262.44 316.15" stroke-linecap="square"/>
+            <path d="M-19.342 113.12L262.44 282.15" stroke-linecap="square"/>
+            <path d="M150.06 309.144v-334.47"/>
+            <path d="M217.06 309.144v-334.47"/>
+            <path d="M83.06 309.144v-334.47"/>
+          </g>
+          <g stroke="url(#transparency-linearGradient-6)" stroke-width="2" mask="url(#transparency-mask-5)">
+            <path class="LogoTransparency-piece TransparencyIcon-pieceTop" d="M82 209.092l68.182 40.91 68.182-40.91v-36.364l-68.182 40.91L82 172.728v36.363z"/>
```

Comment:
Is `.TransparencyIcon-pieceTop` being used for anything. I did a quick search and found no reference to it